nodeUpdate.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
file naming here could be in line with the one for dockerOs. it's essentially the same purpose... (see related comment above)